default/policy.yaml

```diff
@@ -13,6 +13,7 @@
 #       value: github.com/enterprise-contract/config//default
 #     ...
 #
+name: Default
```

Comment:
Does this require https://github.com/enterprise-contract/enterprise-contract-controller/pull/186 ? I think not, it might just mean the name gets dropped.

Reply:
Yes you are correct, we cannot use this until https://github.com/enterprise-contract/enterprise-contract-controller/pull/186 is merged (now is) and propagated. I'll convert this to draft till we see that version in use